[DataGridPremium] AI Assistant (#16992)
mui/mui-x: docs/data/data-grid/components/ai-assistant-panel/ai-assistant-panel.md

---
title: React Data Grid - AI Assistant Panel component
productId: x-data-grid
components: AiAssistantPanelTrigger
packageName: '@mui/x-data-grid-premium'
githubLabel: 'component: data grid'
---

# Data Grid - AI Assistant Panel [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan') 🚧

<p class="description">Customize the Data Grid's AI assistant panel.</p>

:::warning
This component is incomplete.

Currently, the AI Assistant Panel Trigger is the only part of the AI Assistant Panel component available.

> Future versions of the AI Assistant Panel component will make it possible to compose each of its parts to create a custom AI assistant panel.

Reviewer: ```suggestion
Future versions of the AI Assistant Panel component will make it possible to compose each of its parts for full customization.
```
